src/tools/rbd/Utils.h

```diff
@@ -110,7 +110,7 @@ int get_formatter(const boost::program_options::variables_map &vm,
 void init_context();
 
 int init(const std::string &pool_name, librados::Rados *rados,
-         librados::IoCtx *io_ctx);
+         librados::IoCtx *io_ctx, bool force = false);
```

Comment:
The parameter name of ```force``` doesn't really align (name-wise) with what it is actually doing behind the scenes which is to force-apply a special flag to the OSDs. Also, assuming there are no issues w/ the librados changes by others, I don't think this is the correct place to do this. Seems like you should be able to set this flag *after* opening the image for the remove actions so you don't need to put it in this common location.